mslib/msui/mscolab.py

```diff
@@ -549,6 +547,9 @@ def authorize(self):
             if r.status_code == 200:
                 constants.MSC_LOGIN_CACHE[self.mscolab_server_url] = (auth[0], auth[1])
                 self.after_authorize(emailid, r)
+            else:
+                self.error_dialog = QtWidgets.QErrorMessage()
+                self.error_dialog.showMessage('Oh no, server authentication were incorrect.')
```

Comment:
This does not fix #849 but maybe something else?
I made a PR before I noticed you already wanted to tackle this issue. Perhaps just copy that part.
https://github.com/Open-MSS/MSS/pull/867

Reply:
done, thx :)